src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,70 @@ Special packages
 ================
 
 BLAS
-----
```

Comment:
These changes will also affect https://github.com/conda-forge/conda-forge.github.io/blob/master/src/user/tipsandtricks.rst#avoiding-the-openblasmkl-dance, right?

Reply:
I'm not sure, it will happen after switching.

Reply:
I checked and there's no openblas-mkl dance anymore. (We need to fix all packages before this is effective though)